### DISCUSSION TITLE
Turkish language must be added

### DISCUSSION THREAD

**Discussion by balcibaris** [IBAX-io/go-ibax#179] [Ideas] [answered]
Most people who use the application are Turkish.  Therefore, Turkish language should also be added.

**Comment by ibaxio** [comment, marked as answer]
The IBAX team is recruiting volunteers to translate the official website. Due to the rapid growth of the Turkish community, Turkish will be uploaded in the next revision.

We also welcome translations from community members in other languages.

Unleash the power of our community and make us stronger.
